src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ActionBar.xaml.cs

```diff
@@ -48,6 +91,16 @@ private void ActionListViewItem_Tapped(object sender, TappedRoutedEventArgs e)
         if (listItem.DataContext is CommandContextItemViewModel item)
         {
             ViewModel?.InvokeItemCommand.Execute(item);
+            MoreCommandsButton.Flyout.Hide();
+            e.Handled = true;
         }
     }
+
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "VS has a tendency to delete XAML bound methods over-agressively")]
```

Comment:
I haven't seen this elsewhere, so I'm not sure what's going on with the PT config that's causing this to be flagged. We should investigate later and follow-up as it's weird we need these.

Reply:
Maybe it's just me configuring the editor code cleanup rules to aggressively? But it definitely constantly deletes any of the XAML bound methods on save. Only way to stop it is like this, or marking the methods as `internal` (which is not _technically_ correct)

It's really just an annoyance, but I agree we should get to the bottom of it